docs/flashing.md

```diff
@@ -38,7 +38,7 @@ dot indent and a white paint mark. The silkscreen may also indicate pin 1.
 
 These can be purchased from many places for around 15 USD. Make sure that the
 one you get has a ROM clip. Here is an example:
-https://www.amazon.com/Organizer-Socket-Adpter-Programmer-CH341A/dp/B07R5LPTYM
+[Amazon.com](https://www.amazon.com/KeeYees-SOIC8-EEPROM-CH341A-Programmer/dp/B07SHSL9X9) [AliExpress.com](https://aliexpress.ru/item/32725360255.html)
```

Comment:
Include the product vendor and name in the link text. Make them a list for readability.